tests/Sulu/Component/Content/Rlp/Mapper/PhpcrMapperTest.php

```diff
@@ -450,6 +450,24 @@ public function testDeleteByPath()
         $this->assertFalse($rootNode->hasNode('test/news-1/sub-2'));
     }
 
+    /**
+     * The deleteByPath method would delete the entire route tree for the given language, if an empty path is passed.
+     */
+    public function testDeleteByPathWithEmptyPath()
```

Comment:
maybe also add a test for null? i would also prefer to combine this three tests to one with a dataProvider!